tests/test_kruskals.py

```diff
@@ -121,3 +121,26 @@ def test_that_direction_is_applied_on_directional_drivers_analysis():
     series = Kruskals.Kruskals(ndarr, arr).driver_score_to_series(True)
 
     assert (series.values < 0).any()
+
```

Comment:
Not given that the others aren't. We can correct in a different PR if you feel strongly